app/models/task.rb

```diff
@@ -5,4 +5,22 @@ class Task < ApplicationRecord
   default_scope { order(order: "asc") }
 
   delegate :project, to: :section
+
+  def actions_count
+    actions_count ||= actions.all.count
```

Comment:
I don't think this memoization will work without assigning to an instance variable (I haven't ever seen it without)

```
def actions_count
    @actions_count ||= actions.all.count
end
```